### DISCUSSION TITLE
Welcome to NIMPHS Discussions!

### DISCUSSION THREAD

**Discussion by Failxxx** [Artelia/NIMPHS#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LiXun-bit** [comment]
hi,I am currently working on fluid rendering and have simulated cylinder flow using OpenFOAM, where I added a passive scalar to simulate smoke. How can I use the concentration data from the completed simulation to render realistic images with NIMPHS and Blender?

**Reply by Failxxx** [reply]
Hello,
I would suggest you to first explore the documentation : https://artelia.github.io/NIMPHS/
Did you succeed to install Blender and the add-on?

Sincerely,
Félix.
